core/tests/protractor.conf.js

```diff
@@ -338,6 +339,33 @@ exports.config = {
       }));
     }
 
+    var ADD_VIDEO_REPORTER = true;
+    var spw = '';
+    var ffmpegArgs = ['-video_size','1024x768','-framerate','25','-f','x11grab','-i','process.env.DISPLAY'] // add -i argument
+
+    if (ADD_VIDEO_REPORTER) {
+      jasmine.getEnv().addReporter({
+        jasmineStarted: function(result){
+          var name = result.fullName + '.mp4';
+          console.log(result.fullName);
```

Comment:
This value appears to be `undefined` ([log](https://app.circleci.com/pipelines/github/krishnarao22/oppia/184/workflows/2644659c-554b-46d6-961e-ff0ff1761492/jobs/921)). Maybe we want to try defining `suiteStarted` and `suiteDone` instead ([docs](https://jasmine.github.io/tutorials/custom_reporter))?

Reply:
Sure! It also looks like the path we generate is missing a slash (between 'protractor-video' and the video name). Could you also add this?